1-exercises/F-array-forEach/exercise.js

```diff
@@ -9,6 +9,17 @@
 
 let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
 
+arr.forEach((element) =>
+  console.log(
+    element % 3 === 0 && element % 5 === 0
```

Comment:
your code works fine but it is better practice to write 3 console logs instead of witing expression inside console.log